### PR TITLE
Add Safari versions for media CSS @rule

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds real values for Safari (Desktop and iOS/iPadOS) for the `media` CSS @rule.  Since there is nothing in the [source code for Safari 2](https://trac.webkit.org/browser/webkit/tags/old/Safari-421/WebCore/khtml/css) for media queries, it doesn't make sense for this to be set to Safari 1.3.  As I have found in #4369, support for pretty much every part of media queries hadn't been introduced until at least Safari 3.

### CODE REVIEW

File: css/at-rules/media.json [hunk 1]
```diff
@@ -914,10 +914,10 @@
                 "version_added": "10.1"
               },
               "safari": {
-                "version_added": "1.3"
+                "version_added": "3"
               },
               "safari_ios": {
-                "version_added": "3.1"
+                "version_added": "3"
```
Comment (foolip) [inline css/at-rules/media.json:920]: Should this be 1 like above?

**Comment by foolip** [comment]
You linked to https://trac.webkit.org/browser/webkit/tags/old/Safari-421/WebCore/khtml/css, did you mean https://trac.webkit.org/browser/webkit/tags/old/Safari-412/WebCore/khtml/css? (421 vs. 412)

Judging by https://caniuse.com/css-mediaqueries there was initial partial support. I think relying on source code for this is not going to be too reliable. Maybe a simple test can be written instead?

**Comment by vinyldarkscratch** [comment]
I did mean to link the earlier tag, but I think that helps show the lack of support!

It appears that CanIUse doesn't track data as far back as Safari 2 as it is (they have Safari 3.1 listed as the first version).  I ran a quick test with a simple media query (`@media screen and (max-width: 700px) {}`) in both Safari 2 and 3, and have confirmed there's no support in Safari 2.

**Review by foolip — APPROVED** [review]

**Review by foolip — APPROVED** [review]
This also matches https://developer.mozilla.org/en-US/docs/Web/API/CSSMediaRule#browser_compatibility, good!

**Pull request merged**